Source/WebKit/UIProcess/API/APIProcessPoolConfiguration.h

```diff
@@ -177,7 +177,7 @@ class ProcessPoolConfiguration final : public ObjectImpl<Object::Type::ProcessPo
     std::optional<bool> m_processSwapsOnNavigationFromClient;
     bool m_processSwapsOnNavigationFromExperimentalFeatures { false };
     bool m_alwaysKeepAndReuseSwappedProcesses { false };
-    bool m_processSwapsOnWindowOpenWithOpener { false };
+    bool m_processSwapsOnWindowOpenWithOpener { true };
```

Comment:
This should probably get its value from the yaml instead of just switching it to true.